src/locale-en.js

```diff
@@ -32,6 +32,10 @@ module.exports = {
         decimal: '.',
         thousands: ',',
         grouping: [3],
-        currency: ['$', '']
+        currency: ['$', ''],
+        year: '%Y',
+        month: '%b %Y',
+        dayMonth: '%b %e',
+        dayMonthYear: '%b %e, %Y'
```

Comment:
`%e` is [padded with a space](https://github.com/d3/d3-3.x-api-reference/blob/master/Time-Formatting.md) for 1-digit days - we should switch back to `%-d`, which has no padding (in `fr.js` as well). That's responsible for the image test failures.